readme.md

```diff
@@ -14,6 +14,8 @@ For instructions on how to use this and the access control features to use it fo
 ## Dependencies
 
 - Microsoft .NET Core 3.1
+- Microsoft Windows
```

Comment:
I would add a clarification that this is only true for the ssl cert generation. You can generate a cert with openssl or some other cross-platform tool.

Reply:
I can say that it's only for the ssl cert generation, but - it's a mandatory requirement, you can't skip it and go run the server after. Would definitely make sense if it was optional, I agree.